templates/studentProfile.phtml

```diff
@@ -12,25 +12,170 @@
     <script defer src="/js/updateStudentProfile.js"></script>
 </head>
 <body>
-<div hidden data-student-id=<?=intval($id);?> id="studentId"></div>
+<div hidden data-student-id=<?= intval($id); ?> id="studentId"></div>
 <div class="container">
     <div class="col-xs-12 registeredApplicants">
         <h1>Profile</h1>
-            
-                    
-                    <?php if($applicant) {
-                        echo \Portal\ViewHelpers\DisplayStudentProfileViewHelper::outputApplicant($applicant);
-                    } else {
-                        echo 'Sorry, profile not found. Please contact the office.';
-                    }?>
-
-
-            <div class="modal-footer">
+        <hr>
+        <section>
+            <h4>Personal Information</h4>
+            <p class="detail">Name: <span><?php echo $applicant->getName(); ?></span></p>
+            <p class="detail">Email: <span><?php echo $applicant->getEmail(); ?></span></p>
+            <p class="detail">Phone Number: <span><?php echo $applicant->getPhoneNumber(); ?></span></p>
+        </section>
+        <hr>
+        <section>
+            <h4>Application Info</h4>
+            <p class="detail">Stage: <span id="stageName"><?php echo $applicant->getStageName(); ?></span></p>
+            <p class="detail" id="stageOptionNameContainer">Stage Option:
+                <span id="stageOptionName"><?php echo($applicant->getStageOptionName() ?? '--'); ?></span></p>
+            <p class="detail">Cohort: <span id="cohortDate"><?php echo $applicant->getCohortDate(); ?></span></p>
+            <p class="detail">Reason for wanting to be a dev:</p>
+            <p id="whyDev"><?php echo $applicant->getWhyDev(); ?></p>
+            <p class="detail">Code Experience:</p>
+            <p id="codeExperience"><?php echo $applicant->getCodeExperience(); ?></p>
+            <p class="detail">Heard About Us: <span id="hearAbout"><?php echo $applicant->getHearAbout(); ?></span></p>
+            <p class="detail" id="eligible">Eligible to study in the UK: <?php echo '<span>'
+                    . \Portal\ViewHelpers\DisplayStudentProfileViewHelper::studentChecks($applicant->getEligible())
```

Comment:
as your using this method multiple times, you should swap it for a use statement at the top, that will shorten the function calls and make the file tidier